content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -438,7 +439,14 @@ Here `v1.8.x` means the "latest patch release of the v1.8 branch".
 
 `${ARCH}` can be one of: `amd64`, `arm`, `arm64`, `ppc64le` or `s390x`.
 
-If using `--feature-gates=CoreDNS=true` image `coredns/coredns:1.0.2` is required (instead of the three `k8s-dns-*` images).
+In Kubernetes versions before 1.11 and if using `--feature-gates=CoreDNS=true`,
+the image `coredns/coredns:1.0.2` is required (instead of the three `k8s-dns-*` images).
+
+In Kubernetes 1.11 and later, you can also list and pull the images using the `kubeadm config images` sub-command:
```

Comment:
Do we need "also" here? Recommend removing unless this item is really additive to the previous item/s.